specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-07-01/machineLearningServices.json

```diff
@@ -0,0 +1,4768 @@
+          "description": "Number of compute nodes which are being prepared."
+        },
+        "unusableNodeCount": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "title": "Unusable node count.",
+          "description": "Number of compute nodes which are in unusable state."
+        },
+        "leavingNodeCount": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "title": "Leaving node count.",
+          "description": "Number of compute nodes which are leaving the amlCompute."
+        },
+        "preemptedNodeCount": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "title": "Preempted node count.",
+          "description": "Number of compute nodes which are in preempted state."
+        }
+      },
+      "description": "Counts of various compute node states on the amlCompute."
+    },
+    "ClusterUpdateProperties": {
+      "properties": {
+        "scaleSettings": {
+          "$ref": "#/definitions/ScaleSettings",
+          "title": "Scale settings.",
+          "description": "Desired scale settings for the amlCompute."
+        }
+      },
+      "description": "The properties of a amlCompute that need to be updated."
+    },
+    "ClusterUpdateParameters": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ClusterUpdateProperties",
+          "description": "The properties of the amlCompute."
+        }
+      },
+      "description": "AmlCompute update parameters."
+    },
+    "ComputeNodesInformation": {
+      "type": "object",
+      "description": "Compute nodes information related to a Machine Learning compute. Might differ for every type of compute.",
+      "discriminator": "computeType",
+      "properties": {
+        "computeType": {
+          "description": "The type of compute",
+          "$ref": "#/definitions/ComputeType"
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The continuation token."
+        }
+      },
+      "required": [
+        "computeType"
+      ]
+    },
+    "AmlComputeNodesInformation": {
+      "description": "Compute node information related to a AmlCompute.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ComputeNodesInformation"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "nodes": {
+              "readOnly": true,
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/AmlComputeNodeInformation"
+              },
+              "description": "The collection of returned AmlCompute nodes details."
+            },
+            "nextLink": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The continuation token."
+            }
```

Comment:
Maybe we should remove this property, because the base type `ComputeNodesInformation` has the definition of `nextLink`.